### ISSUE TITLE
Problem with the Search functionality

### ISSUE THREAD

**Issue by amyfebryan** [JagratiVerma1408/VideoStreamingapp#1]
Hello sir, I found a problem with the Search functionality. The search feature doesn't work properly when typing a title to search for it comes up with garbled resultsw.

**Comment by JagratiVerma1408** [comment]
yes the feature is only applied to the categories like tv , kids . if you want you can optimize the code for moview title as well.

**Comment by amyfebryan** [comment]
Oh, I see. But I don't really understand programming languages, all I can do is add and tweak.
